contractcourt/commit_sweep_resolver.go

```diff
@@ -57,6 +57,9 @@ type commitSweepResolver struct {
 	// leased channel.
 	leaseExpiry uint32
 
+	// chanType denotes the type of channel the contract belongs to.
+	chanType channeldb.ChannelType
```

Comment:
I think this can either be accessed via `commitSweepResolver.commitResolution.SelfOutputSignDesc` or `contractResolverKit.ResolverConfig.ChannelArbitratorConfig.Channel` so we don't keep the same info in the resolvers again.

Reply:
See above about the closing channels thing.